pyaedt/edb_core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,1700 @@
+
+        Returns
+        -------
+        str
+        """
+        return self.adaptive_settings.MaxRefinement
+
+    @max_refinement.setter
+    def max_refinement(self, value):
+        self.adaptive_settings.MaxRefinement = value
+        self._parent._update_setup()
+
+    @property
+    def max_refine_per_pass(self):
+        """Maximum refinement per pass.
+
+        Returns
+        -------
+        str
+        """
+        return self.adaptive_settings.MaxRefinePerPass
+
+    @max_refine_per_pass.setter
+    def max_refine_per_pass(self, value):
+        self.adaptive_settings.MaxRefinePerPass = value
+        self._parent._update_setup()
+
+    @property
+    def min_passes(self):
+        """Minimum passes.
+
+        Returns
+        -------
+        int
+        """
+        return self.adaptive_settings.MinPasses
+
+    @min_passes.setter
+    def min_passes(self, value):
+        self.adaptive_settings.MinPasses = value
+        self._parent._update_setup()
+
+    @property
+    def save_fields(self):
+        """Save Fields.
+
+        Returns
+        -------
+        bool
+        """
+        return self.adaptive_settings.SaveFields
+
+    @save_fields.setter
+    def save_fields(self, value):
+        self.adaptive_settings.SaveFields = value
+        self._parent._update_setup()
+
+    @property
+    def save_rad_field_only(self):
+        """Save radiated fields only.
+
+        Returns
+        -------
+        bool
+        """
+        return self.adaptive_settings.SaveRadFieldsOnly
+
+    @save_rad_field_only.setter
+    def save_rad_field_only(self, value):
+        self.adaptive_settings.SaveRadFieldsOnly = value
+        self._parent._update_setup()
+
+    @property
+    def use_convergence_matrix(self):
+        """Use convergence matrix.
+
+        Returns
+        -------
+        bool
+        """
+        return self.adaptive_settings.UseConvergenceMatrix
+
+    @use_convergence_matrix.setter
+    def use_convergence_matrix(self, value):
+        self.adaptive_settings.UseConvergenceMatrix = value
+        self._parent._update_setup()
+
+    @property
+    def use_max_refinement(self):
+        """Use Max refinement.
+
+        Returns
+        -------
+        bool
+        """
+        return self.adaptive_settings.UseMaxRefinement
+
+    @use_max_refinement.setter
+    def use_max_refinement(self, value):
+        self.adaptive_settings.UseMaxRefinement = value
+        self._parent._update_setup()
+
+    @pyaedt_function_handler()
+    def add_adaptive_frequency_data(self, frequency, max_num_passes=10, max_delta_s=0.02):
+        """Add a Frequency Data setup.
+
+        Parameters
+        ----------
+        frequency : str, float
+            Frequency with units or float frequency (in Hz).
+        max_num_passes : int, optional
+            Maximum number of passes. Default is ``10``.
+        max_delta_s : float, optional
+            Maximum Delta S. Default is ``0.02``.
```

Comment:
```suggestion
            Maximum delta S. The default is ``0.02``.
```
I don't think the "D" in "Delta" needs to be capitalized.